### ISSUE TITLE
Copied preprocessor stage cannot possibly work for single data samples (Scale by SD e.g.)

### ISSUE THREAD

**Issue by enjoysmath** [biolab/orange3#3219]
[Here's a screenshot of Orange Canvas 3.15](https://screenshot.net/npqjyck)

What's wrong is the Preprocessor stage to my Neural Net training has a Normalization step.   But duplicating the preprocessor for the one-at-a-time prediction flow and the features all vanish to zero because of scaling or w/e by the preprocessor.

How do you use the configuration of a Preprocessor and apply to exactly one sample, while the Preprocessor's Mean / SD are computed from the training dataset?

**Comment by ajdapretnar** [comment]
Never use Preprocess twice. Orange automatically appplies the preprocessor to the test data. Here is a simple example:
<img width="637" alt="screen shot 2018-08-31 at 14 31 36" src="https://user-images.githubusercontent.com/12524972/44912509-b1eb0000-ad2a-11e8-9717-4fc9fc8e7f34.png">

I highly recommend reading the documentation, too.

**Comment by enjoysmath** [comment]
@ajdapretnar Thank you, very much!

Where does it say that in this documentation?

https://docs.orange.biolab.si/3/visual-programming/widgets/data/preprocess.html

**Comment by ajdapretnar** [comment]
@enjoysmath Wow, sorry, I was sure this was in the docs for Prediction! I'll fix it right away and put it in Preprocess as well.
